bigquery/gcloud/aio/bigquery/bigquery.py

```diff
@@ -209,22 +252,14 @@ def _make_load_body(
         body = self._make_insert_body(
             rows, skip_invalid=skip_invalid, ignore_unknown=ignore_unknown,
             insert_id_fn=insert_id_fn or self._mk_unique_insert_id)
-        payload = json.dumps(body).encode('utf-8')
-
-        headers = await self.headers()
-        headers.update({
-            'Content-Length': str(len(payload)),
-            'Content-Type': 'application/json',
-        })
-
-        s = AioSession(session) if session else self.session
-        resp = await s.post(url, data=payload, headers=headers, params=None,
-                            timeout=timeout)
-        return await resp.json()
+        return await self._post_json(url, body, session, timeout)
 
     # https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/insert
```

Comment:
Would you please add a comment linking to `https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationLoad` on the next line?